project/static/css/styles.css

```diff
@@ -1,145 +1,431 @@
+/********** Variables **********/
+
 :root {
-    --light-bg-color: #ffffff;
-    --dark-bg-color: #2e2e2d;
+    --light-color: #ffffff;
+    --primary-color: #2e2e2d;
+    --dark-color: #000000;
     --accent-color: #ffd900;
     --dark-text-color: #2e2e2d;
     --light-text-color: #ffffff;
     --neutral-10: #dedede;
+    --neutral-50: #535353;
     --primary-font: 'Montserrat', sans-serif;
     --secondary-font: 'Raleway', sans-serif;
-    --text-size: 18px;
+    --text-size: 16px;
 }
 
+
+
+/********** Basics **********/
+
 html, body {
-    background-color: var(--light-bg-color);
+    background-color: var(--light-color);
     color: var(--dark-text-color);
     height: 100%;
     font-family: var(--secondary-font);
+    font-size: var(--text-size);
 }
 
-a {
-    text-decoration: none;
+a, .link{
     color: var(--dark-text-color);
+    background:
+        linear-gradient(
+            to bottom, var(--accent-color) 0%,
+            var(--accent-color) 100%
+            );
+        background-position: 0 100%;
+        background-repeat: repeat-x;
+        background-size: 4px 4px;
+    transition: background-size .2s;
+    text-decoration: none;
 }
 
-a:visited {
+a:hover, .link:hover{
```

Comment:
When you go over the headers linksm such as Admin or login/logout section, those links have primary-light font color. Wouldn't it be better if we use on the hover dark-primary font-color?

Reply:
Could be, but this css applies throughout the site.